example-chain-basic-actor/pyproject.toml

```diff
@@ -22,10 +22,13 @@ pytest-mock = "3.14.0"
 pytest-cov = "6.0.0"
 pylint = "3.3.4"
 pylintfileheader = "1.0.0"
-black = "25.01.0"
-isort = "6.0.0"
+black = "^24.10.0"
```

Comment:
Intentional downgrade to align with other modules

Reply:
Pon estas dependencias en todos los módulos. Tienes que ejecutar isort y black y subir los cambios.

pytest = "8.3.5"
pytest-asyncio = "0.25.3"
pytest-mock = "3.14.0"
pytest-cov = "6.0.0"
pytest-alembic = "0.11.1"
pylint = "3.3.5"
pylintfileheader = "1.0.0"
black = "25.01.0"
isort = "6.0.1"
mypy = "1.15.0"